release/release-process.md

```diff
@@ -0,0 +1,140 @@
+---
+title: Release Process
+---
+<!-- TODO -->
+# How to Update Version
+
+<!-- TODO -->
+# How to Tag
+
+<!-- TODO -->
+# Test Process
+
+<!-- TODO -->
+# Build Process
+
+
+# Publish to Maven Central
+
+## Account Setup
+
+Steps taken from Maven Release process outlined by [sonatype.org](http://central.sonatype.org/) and summarized below.
+
+1. Request access to publish to com.twitter by creating account and requesting access in [sonatype.org](https://issues.sonatype.org)
+2. Twitter contact needed to confirm access -[see example](https://issues.sonatype.org/browse/OSSRH-22297)
+3. Install local GPG/PGP to sign files by following [sonatype instructions](http://central.sonatype.org/pages/requirements.html#sign-files-with-gpgpgp) 
+4. Publish public key [see example](http://central.sonatype.org/pages/working-with-pgp-signatures.html#distributing-your-public-key)
+5. Confirm public key is published [see example](http://pgp.mit.edu/) note leading `0x` required for hex keys)
+
+# Release Steps
+
+Follow [Manual Staging Bundle Creation and Deployment](http://central.sonatype.org/pages/manual-staging-bundle-creation-and-deployment.html) by signing files manually, see steps summarized below.
+
+Three artifacts are required for each version: `heron-api`, `heron-storm`, and `heron-spi`
+Example POM template files required located in `heron/release/maven/`
+
+<!-- TODO refer to release "build process" instructions, not download  -->
+<!-- TODO update for "heron-spi" artifact when available -->
+### Step 1 - Download and install `heron-api-install-{VERSION}-{DIST}.sh` from [github releases](https://github.com/twitter/heron/releases)
+
+Example for version `0.14.0` installed using `--user`
+```
+$ wget https://github.com/twitter/heron/releases/download/0.14.0/heron-api-install-0.14.0-darwin.sh
+$ chmod +x heron-api-install-0.14.0-darwin.sh
+$ ./heron-api-install-0.14.0-darwin.sh --user
+```
+Jars are installed into `~/.heronapi`
```

Comment:
@lucperkins @kramasamy periods added for consistency.  problem sentence modified to "After running the above install script with `--user` flag, the `heron-api` artifacts are installed into the `~/.heronapi` directory."